packages/client/utils/isViewerTyping.ts

```diff
@@ -0,0 +1,6 @@
+const isViewerTyping = () => {
+  const activeElement = document.activeElement as HTMLElement
+  return activeElement.contentEditable === 'true'
+}
```

Comment:
I'm not sure about this casting. I haven't found a better way thought so I'd appreciate suggestions if there's a way to avoid casting 🙏

Reply:
yep, having to cast Element to HTMLElement is the cleanest way i've found